contracts/infrastructure/DappRegistry.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.6.12;
+
+import "./IAuthoriser.sol";
+import "./dapp/IFilter.sol";
+import "./base/Owned.sol";
+import "./storage/Storage.sol";
+
+contract DappRegistry is IAuthoriser, Storage, Owned {
+
+    bytes32 constant internal ARGENT_DEFAULT = keccak256("argent_default");
```

Comment:
No need for the keccak IMO. Using the raw string for ARGENT_DEFAULT (and other registry ids) will make debugging easier and saves about 1k of gas when calling `toggle()` thanks to the reduction of non-zero calldata bytes.